r-package/grf/tests/testthat/test_causal_forest.R

```diff
@@ -155,13 +155,13 @@ test_that("predictions are invariant to scaling of the sample weights.", {
 })
 
 test_that("IPCC weighting in the training of a causal forest with missing data improves its complete-data MSE.", {
```

Comment:
So here -- I'm not sure it's really possible to estimate anything with a forest n = 40 and p = 20. So I'm rather skeptical of this change. If this one is tricky, can we leave this one test as is, and follow-up with a separate PR?